### PR TITLE
Feature/iiwa 45

### PR DESCRIPTION
**Pull request by Xiscoder**
Added a class, made license bar graph endpoint, commented some stuff

### CODE REVIEW

**Comment by charkour** [comment]
This is a subtask. So you'll first need to merge it with your story branch.

File: src/backend-api/Controllers/CostController.cs [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+        public CostController(ITInventoryDBContext context)
+        {
+            _context = context;
+        }
+
+        /* GET: api/Cost/Dashboard
+         * Returns [
+         *          Program Cost Per Year,
+         *          Plugin Cost Per Year
+         *         ] of all the programs and their plugins in our database
```
Comment (mtleonard120) [inline src/backend-api/Controllers/CostController.cs]: Change this fella to an object

File: src/backend-api/Controllers/ProgramsController.cs [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+            //Loop through every program in the distinct programs list
+            foreach (var prog in DistinctUsefulPrograms)
+            {
+                //First lamda counts all the programs in the useful program list where the name is the same as the 
+                //name in the distinct programs list
+                var CountProgOverall = UsefulProgramsList.Where(x => x.ProgramName == prog.ProgramName).Count();
+                //Second lamda counts all the programs in the useful program list where the name is the same as the 
+                //name in the distinct programs list and where the license is being used
+                var CountProgInUse = UsefulProgramsList.Where(x => x.ProgramName == prog.ProgramName && x.EmployeeID != null).Count();
+                //adding all the necessary returnables(is that a word?)
```
Comment (mtleonard120) [inline src/backend-api/Controllers/ProgramsController.cs]: lol

File: src/backend-api/Controllers/ProgramsController.cs [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+            // First list removes programs that are not licenses and that are deleted
+            var UsefulProgramsList = _context.Program.Where(x => x.IsLicense == true && x.IsDeleted == false);
+            //Second List takes the previous list and makes it distinct
+            var DistinctUsefulPrograms = UsefulProgramsList.GroupBy(x => x.ProgramName).Select(x => x.FirstOrDefault());
+
+
+            //Loop through every program in the distinct programs list
+            foreach (var prog in DistinctUsefulPrograms)
+            {
+                //First lamda counts all the programs in the useful program list where the name is the same as the
```
Comment (mtleonard120) [inline src/backend-api/Controllers/ProgramsController.cs]: Just an fyi, it's lambda :)

**Pull request merged**